gdbinit.py

```diff
@@ -34,7 +42,13 @@ def calculate_hash(file_path):
 
 
 def run_poetry_install(dev=False):
-    command = ["poetry", "install"]
+    check_poetry()  # Check if poetry executable exists in PATH
```

Comment:
`check_poetry` seems to do essentially what you've added below. I think we can remove it.

Reply:
Okay sure.